scripts/factoids.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(bot) {
       var re = /(.+?)\s(is|are)\s(.+)/;
       var matches = re.exec(text);
       if (!bot.helpers.utils.empty(matches, 1) && !bot.helpers.utils.empty(matches, 2)) {
-        bot.brain.upsertToCollection('factoids', {key: matches[1], channel: to}, {key: matches[1], channel: to, factoid: matches[3], is_are: matches[2]});
+        bot.brain.upsertToCollection('factoids', {key: matches[1].toLowerCase(), channel: to}, {key: matches[1].toLowerCase(), channel: to, factoid: matches[2]});
```

Comment:
Actually, now that I look at this, I think we can also simplify, taking advantage of the fact that when the regex matches, we're guaranteed to have something in `matches[1]`, `matches[2]` and `matches[3]` (b/c of "+" quantifier).  Something like:

```js
if (matches = text.match(/(.+?)\s(is|are)\s(.+)/i)) {
  bot.brain.upsertToCollection('factoids', {key: matches[1].toLowerCase(), channel: to}, {key: matches[1].toLowerCase(), channel: to, is_are: matches[2].toLowerCase(), factoid: matches[3]});
  bot.irc.say(to, nick + ': Okay!');
}

That will also allow the regexp to match case insensitive, so that it'll take "is", "Is", "IS", "ARE", etc.